lib/functions/filedir/function.makeCorrectDir.php

```diff
@@ -27,9 +27,9 @@
 function makeCorrectDir($dir) {
 
 	if (version_compare("5.4.6", PHP_VERSION, ">")) {
-		assert('is_string($dir) && strlen($dir) > 0 /* $dir does not look like an actual folder name */');
```

Comment:
the comment was part of the assertion on purpose as there was no description parameter until 5.4.8 and so the comment would be output as well

Reply:
> the comment was part of the assertion on purpose as there was no description parameter until 5.4.8 and so the comment would be output as well

Ah, that explains why this was in ' all the time. This should therefore be fine now. I tested the else clause on my server and, at least for 7.2 it is now fine. Did not find any errors after that.

Reply:
So, if you could change that back for <=5.4.6 I'd gladly merge